src/config.ts

```diff
@@ -145,10 +145,22 @@ export interface VLOnlyConfig<ES extends ExprRef | SignalRef> {
   fieldTitle?: 'verbal' | 'functional' | 'plain';
 
   /**
-   * D3 Number format for guide labels and text marks. For example `"s"` for SI units. Use [D3's number format pattern](https://github.com/d3/d3-format#locale_format).
+   * If numberFormatType is not specified or is `"format"`,
```

Comment:
```suggestion
   * If `numberFormatType` is not specified or is `"format"`,
```

Reply:
I'm very confused by the special string `"format"` here. Where is this coming from?